src/main/resources/messages.properties

```
@@ -0,0 +1,8 @@
+#Email
+email.invalid=Email invalido, por favor insira um email valido!
+email.not.null=O Email nao pode ser vazio!
+
+#Password
+password.not.null.or.blank=A senha nao pode ser nula ou ter apenas espaço
+password.short=A senha tem que ter mais do que 6 caracteres
```

aqui poderia ser:

```suggestion
password.short=A senha tem que ter mais do que {min} caracteres
```
passando o min, o texto fica dinamico, caso vc altere o argumento na anotação, a mensagem nao quebra